README.md

```diff
@@ -8,7 +8,11 @@ Open the command pallet and type `>Lua Format` to run the formatter.
 
 ## Requirements
 
-Formatter only works with lua version 5.1.
+Formatter only works with Lua version 5.1. In addition the complete path to Lua must be specified in the configs such as:
+```
+vscode-metalua-formatter.luaPath = '/usr/local/bin/lua'
```

Comment:
Everyone will be getting their Lua 5.1 through `lenv`, so a more useful path would be `/home/<user>/.lenv/lua/5.1.5/bin/lua` for Linux and `/Users/<user>/.lenv/lua/5.1.5/bin/lua` for macOS.